Source/Charts/Highlight/ChartHighlighter.swift

```diff
@@ -136,7 +138,7 @@ open class ChartHighlighter : NSObject, IHighlighter
         {
             if axis == nil || high.axis == axis
             {
-                let cDistance = usingXDistance ? getDistance(x1: x, y1: y, x2: high.xPx, y2: high.yPx) : getXDistance(x1: x, x2: high.xPx)
+                let cDistance = distanceCalculation(x, y, high.xPx, high.yPx)
```

Comment:
This is the crux of the refactor, for other folks reading. The calculation method is now injected through `ChartHighlighter.distanceCalculation` versus sending a boolean through to the method (which could get unwieldy down the line if we introduce more).